doc/usecases/translation.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Translation
+
+## Summary
+
+- **Scope:** Translation
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can translate the app's interface and text into English or German
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and wants to change the language of the interface.
+- **Main success scenario:**
+  The user successfully translates the app's interface and text into English or German by selecting the desired language from the app's menu.
```

Comment:
Can you additionally guess the language from the browser infos? (But language should be changeable in the app's settings anyway.)

Reply:
Should be possible, will add it to the decision